troposphere/cognito.py

```diff
@@ -205,6 +205,7 @@ class UserPool(AWSObject):
         'EmailVerificationMessage': (basestring, False),
         'EmailVerificationSubject': (basestring, False),
         'LambdaConfig': (LambdaConfig, False),
+        'EnabledMfas': (basestring, False),
```

Comment:
Minor fixup, please move this line up one to alphabetize the properties. Thanks!

Reply:
updated. thanks.